### ISSUE TITLE
DOC: The link in the documentation for the graph operator is broken.

### ISSUE THREAD

**Issue by derbuihan** [Qiskit/retworkx#594]
### Information

- **retworkx version**: 0.11.0
- **Python version**:
- **Rust version**:
- **Operating system**:

### What is the current behavior?

The Graph Operator link within https://qiskit.org/documentation/retworkx/api.html is broken.
Therefore I cannot read the documentation for retworkx.complement(), retworkx.union(), and retworkx.cartesian_product().

### Steps to reproduce the problem

https://qiskit.org/documentation/retworkx/apiref/retworkx.complement.html
https://qiskit.org/documentation/retworkx/apiref/retworkx.union.html
https://qiskit.org/documentation/retworkx/apiref/retworkx.cartesian_product.html

**Comment by mtreinish** [comment]
Good catch, it looks like it was an oversight in #544 where we moved from `stubs/` to `apiref/` for the auto generated api documentation.  This is easy to fix and just requires removing the last reference to `stubs` in the `api.rst`.

**Comment by mtreinish** [comment]
This is fixed on the development version of the documentation: https://qiskit.org/documentation/retworkx/dev/apiref/retworkx.complement.html#retworkx.complement after the next retworkx release (whenever that is) the released documentation version will be updated.
